src/controllers/words.js

```diff
@@ -64,8 +69,21 @@ export const getWords = async (req, res, next) => {
       dialects,
       examples,
     };
-    let query = !strict ? searchIgboTextSearch(searchWord, isUsingMainKey) : strictSearchIgboQuery(searchWord);
-    const words = await searchWordUsingIgbo({ query, ...searchQueries });
+    let query = !strict
+      ? searchIgboTextSearch(
+        searchWord
+          .split('')
+          .filter((c) => c !== '"' && c !== "'" && !(searchWord.match(/'.*'/) || searchWord.match(/".*"/)))
```

Comment:
The logic that we want to implement with quotes is to prioritize all database queries to be in English if quotes are present in the keyword.
What's happening here is that you are still prioritizing searching with Igbo whether or not quotes are present.

You should:
1. Check to see if the `searchWord` is wrapped in quotes
2. If the `searchWord` is wrapped in quotes, then you want to call `searchEnglishRegexQuery(...)` first, near the top of this function. This means that `words` should should be moved to the top of the function.
3. If there the `searchWord` has no quotes, then you want to go about the original logic of checking the `strict` query ... (but you have to remove the quotes)
4. If `words` turns out to be an empty array, then we fallback to then search with the keyword as English